src/label/Label.ts

```diff
@@ -45,12 +44,6 @@ export const LabelBase = ThemeableMixin(WidgetBase);
 
 @theme(baseCss)
 export default class Label extends LabelBase<LabelProperties>  {
-	@onPropertiesChanged()
-	protected onPropertiesChanged(event: PropertiesChangeEvent<this, LabelProperties>) {
```

Comment:
In fact @bitpshr this should just work as it is now; I have made the registry optional and the registry mixin code should never be invoked if no registry property is passed.